cocos/asset/assets/texture-cube.ts

```diff
@@ -215,7 +215,8 @@ export class TextureCube extends SimpleTexture {
             _forEachFace(faceAtlas, (face, faceIndex) => {
                 ctx.clearRect(0, 0, imageAtlasAsset.width, imageAtlasAsset.height);
                 const drawImg = face.data as HTMLImageElement;
-                ctx.drawImage(drawImg, 0, 0);
+                // NOTE: on OH platform, drawImage only support ImageBitmap and PixelMap type, so we mark drawImg as any.
+                ctx.drawImage(drawImg as any, 0, 0);
```

Comment:
support -> supports.

Reply:
there is no problem here, because the RenderingContext is implemented in our own Native

we only need to fix the type issue here

Reply:
Changing to any causes an eslint error:

Unsafe argument of type `any` assigned to a parameter of type `CanvasImageSource`.